flutter/example/lib/main.dart

```diff
@@ -74,8 +74,7 @@ Future<void> setupSentry(
       options.addIntegration(LoggingIntegration(minEventLevel: Level.INFO));
       options.sendDefaultPii = true;
       options.reportSilentFlutterErrors = true;
-      options.attachScreenshot = true;
-      options.screenshotQuality = SentryScreenshotQuality.low;
```

Comment:
Was this intentional or by accident?

See my comment [here](https://github.com/getsentry/sentry-dart/pull/2361#discussion_r1842041923)

Reply:
intentional, I was thinking we should be testing the default value for quality when testing manually, so that it's the same as what users see by default.